index.js

```diff
@@ -1,58 +1,66 @@
 'use strict';
 
-var buildLiveReloadPath = require('clean-base-url');
+const buildLiveReloadPath = require('clean-base-url');
+const VersionChecker = require('ember-cli-version-checker');
 
 module.exports = {
   name: 'live-reload-middleware',
 
   contentFor: function(type) {
-    var liveReloadPort = process.env.EMBER_CLI_INJECT_LIVE_RELOAD_PORT;
-    var baseURL = process.env.EMBER_CLI_INJECT_LIVE_RELOAD_BASEURL;
+    let liveReloadPort = process.env.EMBER_CLI_INJECT_LIVE_RELOAD_PORT;
+    let baseURL = process.env.EMBER_CLI_INJECT_LIVE_RELOAD_BASEURL;
 
     if (liveReloadPort && type === 'head') {
       return '<script src="' + baseURL + 'ember-cli-live-reload.js" type="text/javascript"></script>';
     }
   },
 
   dynamicScript: function(options) {
-    var liveReloadOptions = options.liveReloadOptions;
+    let liveReloadOptions = options.liveReloadOptions;
     if (liveReloadOptions && liveReloadOptions.snipver === undefined) {
       liveReloadOptions.snipver = 1;
     }
 
-    var liveReloadPath = buildLiveReloadPath(options.liveReloadPrefix) || '/';
+    let liveReloadPath = buildLiveReloadPath(options.liveReloadPrefix) || '/';
     let liveReloadPort;
+    let path = '';
     if (options.liveReloadPort !== options.port) {
       liveReloadPort = options.liveReloadPort
     }
+    if (options.isLatestEmber && options.liveReloadPrefix) {
+      path = `&path=${options.liveReloadPrefix}/livereload`;
+    }
     return `(function() {${liveReloadOptions ? "\n  window.LiveReloadOptions = " + JSON.stringify(liveReloadOptions) + ";" : ''}
   var srcUrl = ${options.liveReloadJsUrl ? "'" + options.liveReloadJsUrl + "'" : null};
   var host= location.hostname || 'localhost';
   var liveReloadPort = ${liveReloadPort};
   var defaultPort = location.protocol === 'https:' ? 443 : 80;
   var port = liveReloadPort || location.port || defaultPort;
+  var path = '${path}';
   var prefixURL = '${liveReloadPort ? "(location.protocol || 'http:') + '//' + host + ':' + " + liveReloadPort : ''}';
-  var src = srcUrl || prefixURL + '${liveReloadPath + 'livereload.js?port='}' + port + '&host=' + host;
+  var src = srcUrl || prefixURL + '${liveReloadPath + 'livereload.js?port='}' + port + '&host=' + host + path;
```

Comment:
> `'&host=' + host + path;`

This seems a little weird? Don't you need to change the end here to:

```
var src = srcUrl || prefixURL + '${liveReloadPath + 'livereload.js?port='}' + port + '&host=' + host + '&path=' + path;
```

Reply:
I moved it [here](https://github.com/SparshithNR/ember-cli-inject-live-reload/blob/path-fix/index.js#L30-#L32) to give a backward compatibility. We can not have `&path` in our URL for older version.
   if (options.isLatestEmber && options.liveReloadPrefix) {
      path = `&path=${options.liveReloadPrefix}/livereload`;
    }

Reply:
Aha! Perfect, thank you for explaining, sorry I missed that.